controller/env/appenv.go

```diff
@@ -407,6 +407,10 @@ func (ae *AppEnv) GetPeerSigners() []*x509.Certificate {
 	return ae.HostController.GetPeerSigners()
 }
 
```

Comment:
We already have GetId(). That's return the token id from the cert, AFAIK, so this should hopefully not be needed.